index.html

```diff
@@ -8,16 +8,19 @@
   <title>My portfolio</title>
   <link rel="stylesheet" href="style.css">
 </head>
-
 <body>
   <header class="header">
     <a href="#" class="logo">my logo</a>
-    <nav class="nav"><img src="./photos/Union.png" alt="menu button"></nav>
-    <ul class="nav-list">
-      <li> <a href="#portifolio">Portifolio</a></li>
-      <li> <a href="#about">About</a></li>
-      <li> <a href="#contact">Contact</a></li>
-    </ul>
+    <nav class="nav"><img id='pop' src="./photos/Union.png" alt="menu button"></nav>
+    <ul class="menu-up nav-list">
+        <li class="close">&times;</li>
```

Comment:
- [ ] Good job with the styles but for this project you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design as much as possible. Please use the correct color and size for your close(X) button and make sure it matches the design on the template.

### Your design 

![image](https://user-images.githubusercontent.com/83803414/161628597-9fa875fa-e4fa-450c-a47b-463f70120209.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/161628630-4a406ac7-122b-4439-9c5a-dffe673c1209.png)